### ISSUE TITLE
As a user I want to have LDPC's small vertex window in the bottom right corner showing you the x, y, z values of the selected vertex. 

### ISSUE THREAD

**Issue by nilsschmidt1337** [nilsschmidt1337/ldparteditor#394]
![vertexwindow](https://user-images.githubusercontent.com/11047164/37539093-08fa876e-2953-11e8-8a4a-3ead9fd1b95b.PNG)

**Comment by nilsschmidt1337** [comment]
The **ControlEditor** widget is what I need:
```
        org.eclipse.swt.custom.ControlEditor editor = new org.eclipse.swt.custom.ControlEditor(canvas);
        // The editor will be a button in the bottom right corner of the canvas.
        // When selected, it will launch a Color dialog that will change the background
        // of the canvas.
        Button button = new Button(canvas, SWT.PUSH);
        button.setText("FooBar..."); //$NON-NLS-1$

        editor.horizontalAlignment = SWT.RIGHT;
        editor.verticalAlignment = SWT.BOTTOM;
        editor.grabHorizontal = false;
        editor.grabVertical = false;
        Point size = button.computeSize(SWT.DEFAULT, SWT.DEFAULT);
        editor.minimumWidth = size.x;
        editor.minimumHeight = size.y;
        editor.setEditor (button);
```

**Comment by nilsschmidt1337** [comment]
The vertex window should not be displayed when the user adds something on the 3D view (it appears spontaneously and covers a large fraction of the view).

**Comment by nilsschmidt1337** [comment]
Closed with commit ccfba86.
